SR2025/2025-02-24-feb-update.md

```diff
@@ -0,0 +1,74 @@
+
+We provide a space for you to work in, with power and internet access, as well as volunteers able to help you with your kits and hands-on guidance with your robot.
+
+There is one final Tech Day left this year, on [**8th March 2025** - at Raspberry Pi Foundation in Cambridge][cambridge-tech-day-march].
+
+There are still plenty of spaces available, so if you're interested in attending please **[sign up][tech-day-signup]**.
+Volunteers may also be around virtually for support, should you not be able to attend the Tech Day in person.
+
+For this Tech Day, the deadline for you to [let us know][tech-day-signup] that you'd like to attend is Friday 28th February 2025.
+
+## Competition
+
+As you are hopefully aware, the [Competition][programme-competition] is a two day event and is the culmination of all your team's hard work over the past six months.
+This year the competition is being held at the University of Southampton on the weekend of [**Saturday 12th April** and **Sunday 13th April**][competition-event].
+
+### Helping you plan
+
+Your robot will compete in a number of league matches, used to seed the knockout stages, as well as having time to make improvements.
+You will have a pit area where you can work on improving your robot and there will be lots of Blueshirt volunteers around to support your team.
+
+We realise organising these kinds of events can be complex for your institution, especially regarding travel and accommodation.
+To aid with the aspects within the venue we have prepared a [risk assessment][competition-risk-assessment] for the event itself.
+Please reach out if there's any additional information we can provide to make this easier.
+
+Guests are welcome in person at the competition and we'll be livestreaming the event so people can follow along at home.
+For those attending in person, the most exciting matches are usually those in the Knockouts on the Sunday afternoon.
+
+At the competition, all robots undergo a safety check.
+This not only confirms that the robot satisfies our safety regulations, but also the rules.
+Some considerations are that the battery is well protected, the power button is accessible and that you have a way to mount the flag which will identify your robot during matches.
+Please make sure you are familiar with both our [design rules][design-rules] and [safety regulations][safety-regulations] which contain the full details.
+
+### Helping us plan
+
+We're working hard getting the event ready for you.
+In order for us to cement our plans, we need to know roughly how large your team is and whether there are any special considerations we need to take for anyone who is coming.
+We understand you might not know exact details right now, so vague guesses are absolutely fine for now!
+You'll be able to edit your response until our shortly before the competition, after which point please email us if anything changes.
```

Comment:
```suggestion
You'll be able to edit your response until the end of March. After this point, please email us if anything changes.
```
Think we should probably say end of March here as I don't know how many people read the top of forms. Possibly add a mailto:teams@ link on the "email us"?

Reply:
Thanks, I just hadn't gotten to this yet. 9730e14

Also https://github.com/srobo/team-emails/pull/209/commits/547c3babcbde5b3248d7379da9aec2d3a36ac0ec